config/obsoletion.yml

```diff
@@ -27,9 +27,3 @@ renamed:
   RSpec/FactoryBot/FactoryClassName: FactoryBot/FactoryClassName
   RSpec/FactoryBot/FactoryNameStyle: FactoryBot/FactoryNameStyle
   RSpec/FactoryBot/SyntaxMethods: FactoryBot/SyntaxMethods
-
-split:
```

Comment:
Maybe keep this commented out?
Is there a better way to remember to do it on a major release?

Reply:
We can set up a milestone in GitHub and add an issue to it.

Reply:
I added [the milestone](https://github.com/rubocop/rubocop-rspec/milestone/2), but didn’t open a issue.